docs/community/faq/faq-users.md

```diff
@@ -2,69 +2,85 @@
+   - `AutoEmulate` takes simple `X` and `y` ndarrays as input, and returns emulator models that can be saved and loaded with `joblib`. All emulators are written as scikit learn estimators, so they can be used like any other scikit learn model in a pipeline.
 
 ## Data Handling
 
-1. What are the best practices for data preprocessing before using AutoEmulate?
+1. What are the best practices for data preprocessing before using `AutoEmulate`?
    <!-- Tips and recommendations on preparing data, including normalisation, dealing with missing values, or data segmentation. -->
+   - The user will typically run their simulation on a selected set of input parameters (-> experimental design) using a latin hypercube or other sampling method. `AutoEmulate` currently needs all inputs to be numeric and we don't support missing data. By default, `AutoEmulate` will scale the input data to zero mean and unit variance, and there's the option to do dimensionality reduction in `setup()`.
 
 2. How does AutoEmulate handle large datasets?
    <!-- Advice on managing large-scale data analyses, potential memory management features, or ways to streamline processing. -->
-
-3. Can I use AutoEmulate for real-time data analysis?
-   <!-- Insights into the software's ability to process data in real-time and any limitations or considerations. -->
+   - `AutoEmulate` is optimised to work with smaller datasets (in the order of hundreds to thousands of samples). Training emulators with large datasets (hundreds of thousands of samples) may currently require a long time and is not recommended. Emulators are created because it's expensive to evaluate the simulation, so we expect most users to have a relatively small dataset.
 
 ## Troubleshooting
 
-1. What common issues might I encounter when using AutoEmulate, and how can I solve them?
+1. What common issues might I encounter when using `AutoEmulate`, and how can I solve them?
    <!-- A list of frequently encountered problems with suggested solutions, possibly linked to a more extensive troubleshooting guide. -->
+   - TODO
 
-2. How can I report a bug or request a feature in AutoEmulate?
+2. How can I report a bug or request a feature in `AutoEmulate`?
    <!-- Instructions on the proper channels for reporting issues or suggesting enhancements, including any templates or information to include. -->
+   - Please open an issue using the [bug issue template](https://github.com/alan-turing-institute/autoemulate/issues/new/choose).
 
 ## Community and Learning Resources
 
-1. Are there any community projects or collaborations using AutoEmulate I can join or learn from?
+1. Are there any community projects or collaborations using `AutoEmulate` I can join or learn from?
    <!-- Information on community-led projects, study groups, or collaborative research initiatives involving AutoEmulate. -->
+   - TODO
```

Comment:
Always good to connect people up.. I wonder if we could mention for folks to reach out if they want invites to the Slack channel. That might be helpful for others, and we could invite them as guests to the channel.

Reply:
Can we invite people from outside the Turing to the Slack channel?

Reply:
I've added a sentence to reach out to you or me for now!